data/caixin

```diff
@@ -0,0 +1 @@
+caixin.com
```

Comment:
1. tracker domain `w.irs03.com` detected: http://w.irs03.com/iwt/a.gif
   consider adding it to ads

2. dynamic advertisement script detected in http://file.caixin.com/webchannel/all/js/dist/channel.js:
   ```javascript
   // funcname: ads_display
   document.writeln('<script type=\'text/javascript\' src=\'https://pinggai' + Math.floor(10 * Math.random()) + '.caixin.com/s?z=caixin&c=' + e + '&slot=' + ran + '&_attr_user=' + t + '&_referrer=' + encodeURIComponent(o) + '\'></script>')
    ```
   consider add `pinggai0.caixin.com`, `pinggai1.caixin.com`, `pinggai2.caixin.com`, ..., `pinggai9.caixin.com` to ads.